rapp_knowrob_wrapper_subclasses_of_topic: /rapp/rapp_knowrob_wrapper/subclasses_of
rapp_knowrob_wrapper_superclasses_of_topic: /rapp/rapp_knowrob_wrapper/superclasses_of
rapp_knowrob_wrapper_is_subsuperclass_of_topic: /rapp/rapp_knowrob_wrapper/is_subsuperclass_of
rapp_knowrob_wrapper_create_instance_topic: /rapp/rapp_knowrob_wrapper/create_instance
rapp_knowrob_wrapper_dump_ontology_topic: /rapp/rapp_knowrob_wrapper/dump_ontology
rapp_knowrob_wrapper_load_ontology_topic: /rapp/rapp_knowrob_wrapper/load_ontology
rapp_knowrob_wrapper_user_instances_from_class_topic: /rapp/rapp_knowrob_wrapper/user_instances_from_class
rapp_knowrob_wrapper_assign_attribute_value: /rapp/rapp_knowrob_wrapper/assign_attribute_value
rapp_knowrob_wrapper_user_instances_of_class: /rapp/rapp_knowrob_wrapper/user_instances_of_class
rapp_knowrob_wrapper_create_ontology_alias: /rapp/rapp_knowrob_wrapper/create_ontology_alias
rapp_knowrob_wrapper_user_performance_cognitve_tests: /rapp/rapp_knowrob_wrapper/user_performance_cognitve_tests
rapp_knowrob_wrapper_create_cognitve_tests: /rapp/rapp_knowrob_wrapper/create_cognitve_tests
rapp_knowrob_wrapper_cognitive_tests_of_type: /rapp/rapp_knowrob_wrapper/cognitive_tests_of_type
rapp_knowrob_wrapper_record_user_cognitive_tests_performance: /rapp/rapp_knowrob_wrapper/record_user_cognitive_tests_performance
rapp_knowrob_wrapper_clear_user_cognitive_tests_performance_records: /rapp/rapp_knowrob_wrapper/clear_user_cognitive_tests_performance_records

